docs/core/tutorials/with-visual-studio-code.md

```diff
@@ -68,7 +68,7 @@ You can also watch a short video tutorial for further setup help on [Windows](ht
 
     ![Selecting .NET Core](media/with-visual-studio-code/selectcore.png)
 
-5. Add a breakpoint to your project by clicking on the **editor margin**, which is the space on the left of the line numbers in the editor, next to line 9.
+5. Add a breakpoint to your project by clicking on the **editor margin**, which is the space on the left of the line numbers in the editor, next to line 9, or move text cursor into line 9 in the editor and then press <kbd>F9</kbd>.
```

Comment:
move text cursor into --> move the text cursor onto
and then press --> and press